[Shadowrun Sixth World] Bugfix: Add 5 new sprite types to main and cu…

SteveForeman opened the pull request:
# Submission Checklist

> [!WARNING]
> Submission Checklist
> Failure to complete this checklist in its entirety will result in your Pull Request being dismissed. If you have any questions, please feel free to create an issue.

> [!NOTE]
> Draft Pull Requests
> If you are unclear about any of the rules regarding the creation of character sheets, or need assistance from the Roll20 team, please feel free to create a Draft PR and request feedback. We'd much rather provide assistance than reject a PR.

<!-- Check these off by replacing the empty space with an 'x' in each of these boxes. If you fail to meet all these criteria, your PR will be rejected. -->

## Pull Request Title

Please format your pull request in the following way: `[Sheet Name] Change Type: Description`. For example: `[D&D5e] New Feature: Adding dragons to dungeons`. 

- [x] The pull request title clearly contains the name of the sheet I am editing.
- [x] The pull request title clearly states the type of change I am submitting (New Sheet/New Feature/Bugfix/etc.).

## Pull Request Content

- [x] The pull request makes changes to files in only one sub-folder.
- [x] The pull request does not contain changes to any json files in the translations folder (translation.json is permitted)
- [x] The pull request does not, _without express prior permission from affected parties_, include any material that could be considered to infringe on a Publisher's intellectual property rights, such as logos, images, rules text or other rules content.

## New Sheet Details

Please complete this section if you are adding a new sheet to the repository.

If you are submitting a new sheet to the repository, please fill in any empty spaces indicated by [  ].

- The name of this game is: [   ]  
  > _(e.g. Dungeons & Dragons 5th Edition, The Dresden Files RPG)_
- The publisher of this game is: [   ]  
  > _(e.g. Wizards of the Coast, Evil Hat)_
- The name of this game system/family is: [   ]  
  > _(e.g. Dungeons & Dragons, FATE)_
- The rules for this game can be found at the following location: [   ]
  > _(e.g. a link to purchase, ISBN, Half-Price Books)_

- [ ] I have followed the [Character Sheets Standards](https://wiki.roll20.net/Building_Character_Sheets#Roll20_Character_Sheets_Repository) when building this sheet.
- [ ] I have authorization from the game's publisher to make this an official sheet on Roll20 with their name attached.

Select EXACTLY One:
- [ ] This game is a traditionally published game.
- [ ] This game is not a traditionally published game.
- [ ] This sheet is for an unofficial fan game, modification to an existing game, or a homebrew system. [  ADD DETAILS HERE  ]

# Changes / Description

<!-- This is an optional step, but detailing the nature of the changes makes it easier for other contributors to track down bugs and fix issues -->

roll20deploy commented:
[Character Sheet Info](https://app.roll20.net/managesheets/Shadowrun%20Sixth%20World/roll20-character-sheets) *Roll20 Internal Use only.*

SteveForeman commented:
[jaculpiii](https://github.com/jaculpiii) is the community maintainer for this sheet, but I didn't see any way to add them to this myself

SteveForeman commented:
Also found & fixed a bug in the Compile Sprite > Sprite Resist > Resist Fade macro series. The Resist Fade was using normal fading resistance (Willpower + Logic) instead of compiling resistance (Willpower + Charisma)

SteveForeman commented:
* Added 'sprite_resist_dv_modifier' for Technoshaman 
* Added a Register Sprite macro chain that properly calculates DV and applies (or skips) compiling-only bonuses

davidroll20 commented:
Hello @SteveForeman! I'm tagging @jaculpiii here to give them some time to look this over before I move it on!

jaculpiii commented:
@davidroll20 It looks good, I don't see any code changes to sheet workers that could blow things up.  I trust @SteveForeman  has tested his layout updates for the sprite stuff.  Steve, in the future if you want to @ me when you submit a merge request I can look pretty quick to lessen any delays.

James

jaculpiii commented:
@davidroll20 Let's hold off on this, I just tested and something in the html is breaking the sheet. I'll try to find some time this week to track down what's breaking the tab selection.
